build/amis/packer/packer.json

```diff
@@ -98,6 +98,27 @@
         "--extra-vars",
         "kubernetes_version={{user `kubernetes_version`}} kubernetes_cni_version={{user `kubernetes_cni_version`}}"
       ]
+    },
+    {
+      "type": "goss",
+      "vars_file": "goss/redhat-vars.yaml",
```

Comment:
Do these get cleaned up after goss runs?

Reply:
By default the plugin puts everything into the `/tmp` directory. This is cleaned at boot time.